app/grandchallenge/evaluation/tasks.py

```diff
@@ -537,3 +548,21 @@ def assign_submission_permissions(*, phase_pk: uuid.UUID):
     )
     for sub in Submission.objects.filter(phase__id=phase_pk):
         sub.assign_permissions()
+
+
+@acks_late_micro_short_task
+def cancel_external_evaluations_past_timeout():
+    Evaluation = apps.get_model(  # noqa: N806
+        app_label="evaluation", model_name="Evaluation"
+    )
+    timeout_threshold = datetime.now() - timedelta(
```

Comment:
This should be `django.utils.timezone.now`. Likely we should add `datetime.now` to banned modules in a separate PR.